geopatterns_demo/random_phrase.py

```diff
@@ -1,6 +1,22 @@
-import loremipsum
+import random
+
+from geopatterns_demo.models import Thesaurus
+
+
+class MakeWord():
+    """Class forms a random word from the transmitted file."""
+
+    def __init__(self, text_file):
+        self.text_file = text_file
+
+    def random_word(self):
+        """Find random word in text file."""
+        with open(self.text_file, 'r') as text:
+            lines = text.readlines()
+            return random.choice(lines).rstrip('\n')
```

Comment:
There is linter problem:

 S311 Standard pseudo-random generators are not suitable for security/cryptographic purposes. [flake8-bandit]
  return random.choice(lines).rstrip('\n')

Reply:
`bandit` has to complain about this but we're not using this for any cryptographic or security purposes, so I guess `random.choice()` is pretty much okay. We can ignore this.

It is also a good practice to add a comment to explain why the `noqa` was added.

```suggestion
            # random.choice() is considered to be insecure by flake8-bandit, but we're not using this for any kind of cryptographic or security related purpose.
            return random.choice(lines).rstrip('\n')   # noqa: S311
```